### PR TITLE
Check subscription check box by default.

### PR DESCRIPTION
**Pull request by gimite**
Also modify subscription form UI text to make it less confusing for users who don't intend to subscribe.

### CODE REVIEW

File: app/resources/subscribe_captcha.html.template [hunk 1]
```diff
@@ -16,12 +16,23 @@
 {% block content %}
 
 <div>
-  <h2>{% blocktrans with person.primary_full_name as full_name %}Subscribe to updates about {{full_name}}{% endblocktrans %}</h2>
-  {% if subscribe_email %}
+
+  {% if context == 'create_person' %}
+    <h2>{% blocktrans with person_record_link_html|safe as full_name %}The record for {{full_name}} has been registered{% endblocktrans %}</h2>
+  {% else %}{% if context == 'add_note' %}
```
Comment (skywhale) [inline app/resources/subscribe_captcha.html.template]: is elseif still not supported?

File: app/subscribe.py [hunk 2]
```diff
@@ -189,3 +196,8 @@ def post(self):
         html = ' <a href="%s">%s</a>' % (url, link_text)
         message_html = _('You have successfully subscribed.') + html
         return self.info(200, message_html=message_html)
+
+    def __get_person_record_link_html(self, person):
+        return '<a href="%s" target=_blank>%s</a>' % (
+            django.utils.html.escape(self.get_url('/view', id=person.person_record_id)),
+            person.primary_full_name)
```
Comment (skywhale) [inline app/subscribe.py]: do you need to escape primary_full_name as well?

**Comment by skywhale** [comment]
LGTM

File: app/resources/subscribe_captcha.html.template (hunk 1 above)
Comment (gimite) [inline app/resources/subscribe_captcha.html.template]: I tried it and it didn't work. I believe we need to update the library to use it.

File: app/subscribe.py (hunk 2 above)
Comment (gimite) [inline app/subscribe.py]: Good point. Done.

**Comment by gimite** [comment]
Thanks for the review. Merged to the default branch.

**Pull request merged**